version.props

```diff
@@ -0,0 +1,10 @@
+<Project>
+  <PropertyGroup>
+    <VersionPrefix>2.1.0</VersionPrefix>
```

Comment:
How does ANCM actually version?

Reply:
We should talk about this. ANCM versions differently than dotnet. I wonder if we can sync them up.